### PR TITLE
type error fix

### PR DESCRIPTION
**Pull request by Tharki-God**
fixes an type error while using use settings with radio item

### CODE REVIEW

**Review by asportnoy — CHANGES_REQUESTED** [review]
If you have to use types and not interfaces, you need to disable the ESLint rule for the appropriate lines

**Comment by Tharki-God** [comment]
> If you have to use types and not interfaces, you need to disable the ESLint rule for the appropriate lines

doesnt show any error to me, why do we need to disable eslint then?

**Comment by lexisother** [comment]
> doesnt show any error to me, why do we need to disable eslint then?

https://github.com/replugged-org/replugged/actions/runs/6115469346/job/16599047576#step:8:12

**Review by asportnoy — APPROVED** [review]

File: src/renderer/modules/components/RadioItem.tsx [hunk 1]
```diff
@@ -13,9 +14,10 @@ interface RadioOptionType {
   tooltipPosition?: "top" | "bottom" | "left" | "right" | "center" | "window_center";
   icon?: React.ComponentType<unknown>;
   collapsibleContent?: React.ReactNode;
-}
+};
 
-interface RadioProps {
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+type RadioProps = {
```
Comment (FedeIlLeone) [inline src/renderer/modules/components/RadioItem.tsx:20]: Changing this is not required.

**Review by FedeIlLeone — CHANGES_REQUESTED** [review]

**Pull request merged**